application/src/main/java/org/thingsboard/server/controller/RuleChainController.java

```diff
@@ -288,24 +257,11 @@ public RuleChain saveRuleChain(
     public RuleChain saveRuleChain(
             @ApiParam(value = "A JSON value representing the request.")
             @RequestBody DefaultRuleChainCreateRequest request) throws ThingsboardException {
-        try {
-            checkNotNull(request);
-            checkParameter(request.getName(), "name");
-
-            RuleChain savedRuleChain = installScripts.createDefaultRuleChain(getCurrentUser().getTenantId(), request.getName());
```

Comment:
**installScripts** service is not used anymore in RuleChainController and can be fully removed.

Reply:
Ok